lib/Twig/Extensions/Extension/Locale.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * This extension provides access to static methods from the \Locale class.
+ *
+ * @author Matthias Pigulla <mp@webfactory.de>
+ */
+class Twig_Extensions_Extension_Locale extends Twig_Extension
+{
+    public function getFunctions()
+    {
+        return array(
+            new Twig_SimpleFunction('locale_primary_language', 'twig_locale_primary_language'),
+        );
+    }
+}
+
+function twig_locale_primary_language($locale = null)
+{
+    $locale = $locale !== null ? $locale : Locale::getDefault();
```

Comment:
this is not a required dependency of the whole package, as it contains several extension. But ext-intl (or the Symfony partial polyfill for the en locale) is indeed a dependency when using this extension, as it is precisely about integrating intl.